blocks/header/header.css

```diff
@@ -735,3 +735,23 @@ li
     display: none;
   }
 }
+
+.product-discovery--search-bar-results__grid {
```

Comment:
FYI I cannot scroll on mobile emulator in chrome:

![image](https://github.com/user-attachments/assets/355eb0b5-e900-4760-96ed-f059af50b8eb)